src/functions/definitions/slack_function_test.ts

```diff
@@ -0,0 +1,85 @@
+import { SlackFunctionDefinition } from "./slack_function.ts";
```

Comment:
A lot of the tests in this file validate `FunctionDefinition` logic, rather than `SlackFunctionDefinition`

Reply:
The abstract base class was no longer required I've removed it and its test file 💯